source/getting-started/autostart-systemd.markdown

```diff
@@ -17,10 +17,10 @@ $ ps -p 1 -o comm=
 
 If the preceding command returns the string `systemd`, you are likely using `systemd`.
 
-If you want Home Assistant to be launched automatically, an extra step is needed to setup `systemd`. You need a service file to control Home Assistant with `systemd`. If you are using a Raspberry Pi with Raspbian then replace the `[your user]` with `pi` otherwise use your user you want to run Home Assistant. `ExecStart` contains the path to `hass` and this may vary. Check with `whereis hass` for the location.
+If you want Home Assistant to be launched automatically, an extra step is needed to setup `systemd`. You need a service file to control Home Assistant with `systemd`. If you are using a Raspberry Pi with Raspbian then replace the `[your user]` and %1 with `pi` otherwise use your user you want to run Home Assistant. `ExecStart` contains the path to `hass` and this may vary. Check with `whereis hass` for the location.
 
 ```bash
-$ su -c 'cat <<EOF >> /etc/systemd/system/home-assistant@[your user].service
+$ sudo su -c 'cat <<EOF >> /etc/systemd/system/home-assistant@[your user].service
```

Comment:
> I don't know why the file was named with the [your user] suffix.

Because this lead to one of the simplest service units and allows one to run HA as the user how installed it including access to the configuration file.